tests/local.rs

```diff
@@ -258,4 +279,10 @@ mod chrome {
     fn double_close_window_test() {
         tester!(close_window_twice_errors, "chrome")
     }
+
+    #[test]
+    #[serial]
```

Comment:
Ah, only the Firefox tests need `#[serial]` due to a bug in the Firefox WebDriver implementation. You do not need it here :)